lib/scrabble.rb

```diff
@@ -1,18 +1,29 @@
 class Scrabble
-
-  def score(word)
-    1
+  attr_reader :point_values
+  def initialize
+    @point_values = {
+     "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+     "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+     "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+     "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+     "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+     "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+     "Y"=>4, "Z"=>10
+   }
   end
 
-  def point_values
-    {
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
-  end
-end
+   def score(word)
+     point_values[word.upcase]
+     points = word.upcase.split(//).map do |letter|
+       point_values[letter]
+     end
+     points.reduce(0,:+)
+   end
+
+   def score_with_multipliers(word, numbers)
+     point_values[word.upcase]
+     points = word.upcase.split(//).map_with_index.map do |letter, index|
+       point_values[letter] * numbers
+   end
```

Comment:
I can see some of your thought path here, it looks like you were started on a good path but got a little lost.  You use map_with_index, where I think you're looking for map.with_index. After you call that you call map on it, but you're already using a .map. 

On line 26 you access the hash, but I think maybe you lost track of what numbers is holding? It should be an array of numbers. So, when you multiply the integer out of the point_values hash times an array, Ruby isn't going to know what to do.  How could you have combined the integer for each letter with the corresponding integer from the numbers array?